ui_testing/testcases/basic_tests/test006_orders.py

```diff
@@ -2578,7 +2589,7 @@ def test076_search_with_test_unit_name(self):
         else:
             test_unit_suggestion_list = Order().create_new_order_get_test_unit_suggetion_list(
                 material_type='', test_unit_name=' ')
-        self.info('checking name field only is displayed')
+        self.info('checking {} field only is displayed'.format(search_by))
         for test_unit in test_unit_suggestion_list:
             self.assertNotIn(':', test_unit)
```

Comment:
you can make use of this method:
https://github.com/modeso-open-source/1lims-automation/blob/4b510411856316f9b3e5ae4715a4d2a9a1d7558a/ui_testing/testcases/basic_tests/test004_testunits.py#L1399
instead of this assertion, as I found that this assertion not correct, It will assert that you can select one option only, but not assert that selected option is name or method